selftests/unit/test_dispatcher.py

```diff
@@ -12,11 +13,20 @@ def test_order(self):
         namespaces = ['avocado.plugins.cli',
                       'avocado.plugins.cli.cmd',
                       'avocado.plugins.job.prepost',
-                      'avocado.plugins.result']
+                      'avocado.plugins.result',
```

Comment:
This is a matter of preference, but I'd prefer an explicit change of the data structure here.  For instance:

```python
namespace_keywords = [('avocado.plugins.cli', {}),
                      ...
                      ('avocado.plugins.resolver', {'config': None'}),
...
```

One of the reasons is that it improves the readability of the code because of the better named variable, and it removes a conditional block.